documentation/modules/con-kafka-connect-authentication.adoc

```diff
@@ -42,17 +45,16 @@ spec:
   # ...
 ----
 
-== SASL based SCRAM-SHA-512 authentication
+To use this method, see xref:proc-configuring-kafka-connect-authentication-tls-{context}[].
 
-To configure Kafka Connect to use SASL-based SCRAM-SHA-512 authentication, set the `type` property to `scram-sha-512`.
-This authentication mechanism requires a username and password.
+== SASL-based SCRAM-SHA-512 authentication
```

Comment:
Same with TLS. We could just refer to https://strimzi.io/docs/operators/master/using.html#type-KafkaClientAuthenticationScramSha512-reference. If so, I think that section would need a bit more detail from here